src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -390,18 +408,20 @@ class SchedulingQueue {
     }
 
     // Cancel any stale requests that the client doesn't need any longer.
-    while (!pending_tasks_.empty() && pending_tasks_.begin()->first < next_seq_no_) {
-      auto head = pending_tasks_.begin();
+    while (!pending_actor_tasks_.empty() &&
+           pending_actor_tasks_.begin()->first < next_seq_no_) {
+      auto head = pending_actor_tasks_.begin();
       RAY_LOG(ERROR) << "Cancelling stale RPC with seqno "
-                     << pending_tasks_.begin()->first << " < " << next_seq_no_;
+                     << pending_actor_tasks_.begin()->first << " < " << next_seq_no_;
```

Comment:
Is ordering only guaranteed for actor tasks?

Reply:
I think so? Let me know if you think that's not the case though :)

Reply:
I am not sure haha. cc @stephanie-wang